### PR TITLE
[DevOps-745] Update Kooper

### PR DESCRIPTION
**Pull request by jchanam**
* Update Kooper to 0.3.0
* Update Kubernetes version to v1.10.5
* Adapt code to Kooper new version
* Proposed tag to 0.3.0
* Update go-redis, logrus, testify, prometheus to latest version

### CODE REVIEW

File: metrics/metrics_test.go [hunk 1]
```diff
@@ -95,7 +97,8 @@ func TestPrometheusMetrics(t *testing.T) {
 
 			// Create the muxer for testing.
 			mx := http.NewServeMux()
-			pm := metrics.NewPrometheusMetrics(path, mx)
+			reg := prometheus.NewRegistry()
+			pm := metrics.NewPrometheusMetrics(path, "redis_operator", mx, reg)
```
Comment (ese) [inline metrics/metrics_test.go]: Pass the constant here `metricsNamespace`

File: .travis.yml [hunk 2]
```diff
@@ -10,8 +10,8 @@ go:
 go_import_path: github.com/spotahome/redis-operator
 
 before_install:
-  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
-  - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.9.4/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+  - curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.28.0/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```
Comment (slok) [inline .travis.yml]: Minikube `v0.28.0` has problems in travis, `v0.25.2` with Kubernetes `1.10.0` version works like a charm

Comment (jchanam) [reply .travis.yml]: Thanks! Changed.

File: metrics/metrics_test.go (hunk 1 above)
Comment (jchanam) [reply metrics/metrics_test.go]: That constant is private in another package (main).

I've changed it to a different non-related namespace to show that it's configurable and that it changes the result of the metrics exposed.

**Pull request merged**